libs/execution/src/lib/constraints/executors/allowlist-constraint-executor.ts

```diff
@@ -27,7 +23,9 @@ export class AllowlistConstraintExecutor implements ConstraintExecutor {
 
     const allowlist = context.getPropertyValue(
       'allowlist',
-      new CollectionValuetype(PrimitiveValuetypes.Text),
+      context.wrapperFactories.ValueType.createCollection(
```

Comment:
The `createCollection` method may be better moved to the `PrimitiveValueTypeProvider`? Because this is not really wrapping an AST node..

Reply:
Implemented in [14fb324](https://github.com/jvalue/jayvee/pull/549/commits/14fb324359cf21bbb4b2bc6599bab7f7c4cc759b) and [c8ac828](https://github.com/jvalue/jayvee/pull/549/commits/c8ac828812c3ad9686f6d3c2172970163e3f5a22)